javascript/javascript-maps.md

```diff
@@ -0,0 +1,96 @@
+# Javascript Maps
+
+## Projected Time
+
+30-45 minutes
+
+## Prerequisites
+
+- [Javascript 6 Lesson](https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-6-object-literals.md)
+
+## Motivation
+
+Map object provided by ES6. Map is a collection of elements where each element is stored as a Key, value pair. Map object can hold both objects and primitive values as either key or value.
+
+## Objectives
+
+**Participants will be able to:**
+
+- Learn maps.
+- Learn Sets.
+- Know the difference between maps and objects.
+- Create and use maps & sets.
+
+## Specific Things To Teach
+
+- Maps.
```

Comment:
Can you be a level more specific here? Ex:
- Maps
  - Syntax
  - Methods
  - Maps and objects compared.
- Sets
 - etc

Reply:
I meant for the changes to be in ‘Specific things to teach’, not objectives.